percy-xcui/Sources/PercyXcui/metadata/Metadata.swift

```diff
@@ -162,4 +168,44 @@ internal class Metadata {
       }
     #endif
   }
+
+  func mapToDeviceStatusBar(identifier: String) -> Int {
+    switch identifier {
```

Comment:
list of devices are very limited, what will happen if someone builds on older device using old code version?